bsyncpy/bsync.py

```diff
@@ -3149,64 +3197,83 @@ class EscalationRate(BSElement):
+        "Association of Energy Engineers Energy Efficiency Practitioner (EEP)",
         "Association of Energy Engineers Performance Contracting and Funding Professional (PCF)",
-        "Association of Energy Engineers Certified Residential Energy Auditor (REA)",
-        "Association of Energy Engineers Certified Building Commissioning Firm Program (CBCF)",
-        "Association of Energy Engineers Certified Green Building Engineer (GBE)",
-        "Association of Energy Engineers Certified Energy Manager (CEM)",
-        "Association of Energy Engineers Certified Energy Auditor (CEA)",
-        "Association of Energy Engineers Certified Building Commissioning Professional (CBCP)",
+        "Association of Energy Engineers Renewable Energy Professional (REP)",
+        "Building Commissioning Association (BCA) Certified Commissioning Professional (CCP)",
+        "Building Commissioning Certification Board (BCCB) Certified Commissioning Professional (CCP)",
         "Building Operator Certification (BOC): Level 1",
         "Building Operator Certification (BOC): Level 2",
+        "Building Owners and Managers Institute (BOMI) International - Facilities Management Administrator (FMA)",
+        "Building Owners and Managers Institute (BOMI) International - High-Performance Sustainable Building Management (BOMI-HP)",
+        "Building Owners and Managers Institute (BOMI) International - Real Property Administrator (RPA)",
+        "Building Owners and Managers Institute (BOMI) International - System Maintenance Administrator (SMA)",
+        "Building Owners and Managers Institute (BOMI) International - System Maintenance Technician (SMT)",
         "Building Performance Institute (BPI) Certification",
-        "Building Performance Institute (BPI): Building Analyst (BA)",
         "Building Performance Institute (BPI): Advanced Home Energy Professional (HEP)",
+        "Building Performance Institute (BPI): Advanced Home Energy Professional - Crew Leader (HEP-CL)",
         "Building Performance Institute (BPI): Advanced Home Energy Professional - Energy Auditor (HEP-EA)",
         "Building Performance Institute (BPI): Advanced Home Energy Professional - Quality Control Inspector (HEP-QCI)",
         "Building Performance Institute (BPI): Advanced Home Energy Professional - Retrofit Installer (HEP-RI)",
-        "Building Performance Institute (BPI): Advanced Home Energy Professional - Crew Leader (HEP-CL)",
+        "Building Performance Institute (BPI): Building Analyst (BA)",
         "Building Performance Institute (BPI): Multifamily Building Analyst",
-        "Residential Energy Services Network (RESNET) Certification",
-        "Residential Energy Services Network (RESNET) - Home Partner",
-        "Registered Architect (RA)",
-        "Refrigerating System Operating Engineer",
-        "High Pressure Boiler Operating Engineer",
         "Certified Commissioning Professional (CCP)",
```

Comment:
Is this not redundant now that there is BCCB CCP and/or BCA CCP?

Reply:
@ThibaultMarzullo, I agree that the generic enumeration is redundant; however, I think that should retain them for backwards compatibility.